stages/org.osbuild.vagrant.meta.json

```diff
@@ -52,6 +52,11 @@
                 "mac_address": {
                   "type": "string",
                   "pattern": "^[a-fA-F0-9]{12}$"
+                },
+                "guest_additions": {
+                  "description": "Whether guest additions are installed. If they are not, then /vagrant is set to rsync",
+                  "type": "boolean",
+                  "default": false
```

Comment:
Would this make more sense to be specifically about the synced folder type?
```
"synced_folder_type": "rsync|vboxfs"
```

That way, we avoid overloading the new option with new behaviour in the future, if we ever need to make more changes based on the existence or absence of the guest additions.

Reply:
Done. I've applied the additional `synced_folders` property only under the virtualbox variant of the schema as otherwise it started getting a bit convoluted and duplicated (`vboxfs` as a value would never be available under `libvirt` as a provider, I tried with `$ref` but it wasn't very nice).